src/common/pagination/Pagination.tsx

```diff
@@ -1,48 +1,71 @@
 import classNames from 'classnames';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import ReactPaginate, { ReactPaginateProps } from 'react-paginate';
 import { IconArrowLeft, IconArrowRight } from 'hds-react';
 
 import Text from '../text/Text';
 import styles from './pagination.module.scss';
 
-export type PaginationProps = Pick<ReactPaginateProps, 'pageCount' | 'onPageChange' | 'initialPage' | 'forcePage'> & {
+export interface PaginationProps {
   className?: string;
-};
+  pageIndex: number;
+  pageCount: number;
+  onPageChange(selectedItem: { selected: number }): void;
+}
 
-const Pagination = ({ className, ...rest }: PaginationProps) => {
+const Pagination = ({ className, pageIndex, pageCount, onPageChange }: PaginationProps) => {
   const { t } = useTranslation();
 
+  const options: JSX.Element[] = [];
+
+  for (let pageIndex = 0; pageIndex < pageCount; pageIndex += 1) {
+    options.push(
+      <option key={pageIndex} value={pageIndex}>
+        {pageIndex + 1}
+      </option>
+    );
+  }
```

Comment:
This seems to shadow the `pageIndex` prop

Reply:
Good catch! Renamed.